modules/admin-ui/src/main/resources/OSGI-INF/event_endpoint.xml

```diff
@@ -13,68 +13,42 @@
     <provide interface="org.osgi.service.cm.ManagedService"/>
   </service>
   <reference bind="setAdminUIConfiguration"
-             cardinality="1..1"
              interface="org.opencastproject.adminui.impl.AdminUIConfiguration"
-             name="adminUIConfiguration"
-             policy="static"/>
+             name="adminUIConfiguration"/>
   <reference bind="setWorkflowService"
-             cardinality="1..1"
              interface="org.opencastproject.workflow.api.WorkflowService"
-             name="workflowService"
-             policy="static"/>
+             name="workflowService"/>
   <reference name="jobService"
              interface="org.opencastproject.adminui.endpoint.JobEndpoint"
-             cardinality="1..1"
-             policy="static"
              bind="setJobService"/>
-  <reference name="seriesService"
```

Comment:
Is this fixing a separate issue?

Reply:
This is just the name which I found to be quite confusing since it's not actually the binding of the series service but the admin interfaces series endpoint. Again, this could technically be a separate, one-line pull request.